packages/web-components/src/dialog-body/dialog-body.styles.ts

```diff
@@ -0,0 +1,124 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase500,
+  fontWeightRegular,
+  fontWeightSemibold,
+  lineHeightBase300,
+  lineHeightBase500,
+  spacingHorizontalS,
+  spacingHorizontalXXL,
+  spacingVerticalL,
+  spacingVerticalS,
+  spacingVerticalXXL,
+} from '../theme/design-tokens.js';
+
+/** Dialog Body styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  /* formerly .root */
+  :host {
+    background: ${colorNeutralBackground1};
+    box-sizing: border-box;
+    display: grid;
+    gap: ${spacingVerticalS};
+    padding: ${spacingVerticalXXL} ${spacingHorizontalXXL};
+    container: dialog-body / inline-size;
+  }
+
+  .title {
+    font-size: ${fontSizeBase500};
+    line-height: ${lineHeightBase500};
+    font-weight: ${fontWeightSemibold};
+    font-family: ${fontFamilyBase};
+    color: ${colorNeutralForeground1};
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    column-gap: 8px;
+
+    /* Sticky header */
+    background: ${colorNeutralBackground1};
+    inset-block-start: 0;
+    margin-block-end: calc(${spacingVerticalS} * -1);
+    margin-block-start: calc(${spacingVerticalXXL} * -1);
+    padding-block-end: ${spacingVerticalS};
+    padding-block-start: ${spacingVerticalXXL};
+    position: sticky;
+    z-index: 1;
+  }
+
+  /* Hide slot if nothing is slotted to remove gap */
+  :not(:has(:is([slot='title'], [slot='title-action']))) .title:not(:has(.title-action)) {
+    display: none;
+  }
+
+  .title:empty {
+    background: pink;
```

Comment:
Was this for debugging? What if someone doesn't want a title?

Reply:
It was for debugging. `:empty` doesn't work unfortunately because there are `<slot>` elements and whitespace in the `div.title` element. Will remove.

Reply:
Ran into this the other day, such a bummer!